mods/cnc/rules/vehicles.yaml

```diff
@@ -660,6 +656,7 @@ STNK:
 		CloakDelay: 90
 		CloakSound: trans1.aud
 		UncloakSound: trans1.aud
+		UncloakOn: Attack, Unload, Infiltrate, Demolish, Dock, Damage, Heal
```

Comment:
I don't think we need `Infiltrate` and `Demolish` here, since Stealth Tanks can't function as spy or lay c4?

Reply:
Just in case? ;)

I'll remove it :P